libminifi/include/io/ClientSocket.h

```diff
@@ -20,10 +20,19 @@
 
 
 #include <cstdint>
+#ifdef WIN32
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+#include <Windows.h>
+#include <WS2tcpip.h>
+#pragma comment(lib, "Ws2_32.lib")
```

Comment:
I'll try to minimize the includes on windows and see if the `#pragma` is necessary after I've verified that my rebase doesn't break tests on Windows.